### ISSUE TITLE
quick links missing

### ISSUE THREAD

**Issue by prosolucoes** [kevinwucodes/gmail-quick-links#8]
I've installed quick links and it worked fine for a few days.
Now it's missing from gmail.

I've already uninstalled and reinstalled but still missing.
Any suggestion?

Thanks
Paulo Soares

**Comment by kevinwucodes** [comment]
Quick Links should show up directly underneath your last label on the left hand-side navigation.  Sometimes you'll see a "label" called "More" with an arrow pointing down.  It should show up directly underneath that.

**Comment by prosolucoes** [comment]
It should be there was it was in the beginning, but now it's missing.
Thanks for your time!

![screenclip](https://user-images.githubusercontent.com/32128979/30690069-e37b6bd4-9eba-11e7-8eaf-0134f300684d.png)

**Comment by kevinwucodes** [comment]
Lets makes sure the extension is fully loaded.  When you open up your console, do you see a message that says "Loaded Gmail Quick Links" ?

To get to the console in Chrome, go to View -> Developer -> JavaScript Console

**Comment by prosolucoes** [comment]
Yes, the message shows up in console. These are the details in index.js:

```
import React, { Component } from 'react'
import { render } from 'react-dom'

import AppContainer from './AppContainer'

import { getGmailLocationToInject } from './config'

const beginReact = accountName => {
  render(
    <AppContainer accountName={accountName}/>,
    document.getElementById('gmailQuickLinksContainer')
  )
}

// there are cases where the gmail left nav panel isn't loaded yet,
// so we need to check for it until it exists before we can proceed
const checkDomElementExist = setInterval(() => {
  if (getGmailLocationToInject()) {
    clearInterval(checkDomElementExist)

    // create a div for our react container to be injected
    const gmailQuickLinksContainer = document.createElement('div')
    gmailQuickLinksContainer.id = "gmailQuickLinksContainer"
    getGmailLocationToInject().append(gmailQuickLinksContainer)

    console.log('Loaded Gmail Quick Links')

    //TODO: what is the person isn't signed in?  Does this crash extension?
    const currentAccountName = document
      .querySelectorAll('a[title^="Google Account"]')[0]
      .title
      .match(/\(([^)]+)\)/)[1]

    //load react
    beginReact(currentAccountName)
  }
}, 200)

```


// WEBPACK FOOTER //
// ./src/index.js

**Comment by kevinwucodes** [comment]
ok, so it does look like the extension is loaded and working.  Maybe the quick links is hidden somehow because you might have some other extensions or some other Gmail Labs turned on?

For the sake of diagnostics, can you try disabling any other Gmail Labs that might be running to see if this lets you see the Quick Links div box?

Also, try doing this in the console

```document.getElementById('gmailQuickLinksContainer')```

If you get "null" returned, something is wrong.  But if you get something like ```<div id="gmailQuickLinksContainer">...</div>``` and then try to mouse hover over that, Chrome should try and reveal where that div is actually located in Gmail.

**Comment by kevinwucodes** [comment]
Fixed in #13

**Comment by prosolucoes** [comment]
Kevin, first of all, thanks, it's working fine again.

Sorry for the delay, but somehow this email thread was deleted, and don't get to my attention.

Today, after a Gmail refresh, it all come back as expected.

Once again, thanks.
